mypy/semanal.py

```diff
@@ -484,6 +484,10 @@ def __init__(
         # Used to pass information about current overload index to visit_func_def().
         self.current_overload_item: int | None = None
 
+        # Used to track whether currently inside an except* block. This helps
+        # to invoke errors when continue/break/return is used inside except * block.
+        self.is_in_except_block: bool = False
```

Comment:
For clarity, I'd name this in a way that indicates this is specific to `except*` blocks. Maybe `inside_except_star_block`?